std/regex/internal/tests.d

```diff
@@ -443,18 +441,18 @@ alias Sequence(int B, int E) = staticIota!(B, E);
                 auto r = ctRegex!(tv[v].pattern, tv[v].flags);
                 auto nr = regex(tvd.pattern, tvd.flags);
                 assert(equal(r.ir, nr.ir),
-                    text("!C-T regex! failed to compile pattern #", a ,": ", tvd.pattern));
+                    text("!C-T regex! failed to compile pattern #", v ,": ", tvd.pattern));
```

Comment:
The number `a` wasn't very useful here anyhow - `v` is the actual number of the test.